app/models/Sprint.php

```diff
@@ -37,6 +38,39 @@ public function getPhabricatorError()
 		return $this->phabricatorError;
 	}
 
+	/**
+	 * Returns a formatted list of dates from sprint_start to sprint_end
+	 *
+	 * @param string $format
+	 * @return array
+	 */
+	public function formatDays($format = 'M j')
+	{
+		return array_map(function($date) use($format)
+		{
+			return date($format, $date);
+		}, $this->getDays());
+	}
+
+	private function getDays()
+	{
```

Comment:
You can make this simpler by doing

```
{
if ( $this->days === null ) $this->days = $this->computeDays();
return $this->days;
}
```

That way you have a method doing the caching and one doing the computation, and not one doing both.